localization/english/CWP/africa/CWP_libya_l_english.yml

```
@@ -0,0 +1,140 @@
+  libya.6.d: "In the face of relentless diplomatic pressure, [SCOPE.sCountry('gaddafi_unification_target_scope').GetName] has taken a momentous step towards embracing unity, deciding to join our distinguished Union. Despite their initial resilience, the unyielding force of diplomatic persuasion has paved the way for their integration."
+  libya.6.f: "There is no state with a democracy except Libya on the whole planet.\n― Muammar Gaddafi"
+  libya.6.a: "The union grows"
+
+  libya.7.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] rejects our Offer"
+  libya.7.d: "In a resolute stand for its sovereignty and independence, [SCOPE.sCountry('gaddafi_unification_target_scope').GetName] firmly declines Colonel Gaddafi's proposition to join his envisioned union. Despite diplomatic pressures and enticing promises, the nation remains steadfast in its refusal, seeing the proposal as a threat to its hard-won autonomy."
+  libya.7.f: "The times of Arab nationalism and unity are gone forever. These ideas which mobilized the masses are only a worthless currency. Libya has had to put up with too much from the Arabs for whom it has poured forth both blood and money.\n― Muammar Gaddafi"
+  libya.7.a: "Traitors!"
+
+  libya.8.t: "[SCOPE.sCountry('gaddafi_unification_target_scope').GetName] rejects our Offer"
+  libya.8.d: "In a tense turn of events, Muammar Gaddafi contemplates a daring coup attempt against the government of [SCOPE.sCountry('gaddafi_unification_target_scope').GetName], following their resounding refusal to join his proposed union. The rejection has sent shockwaves through diplomatic circles, exacerbating already strained relations between the neighboring nations."
+  libya.8.f: "I staged a coup d’état and the workers staged the revolution, occupying the factories, becoming partners instead of employees, eliminating the monarchic administration and setting up people’s committees. In short, they liberated themselves. Students did the same. Today in Libya, only the people count.\n― Muammar Gaddafi"
+  libya.8.a: "Attempt to overthrow their government"
+  libya.8.b: "We want nothing to do with traitors"
+
+  libya.9.t: "Gaddafis Influence spreads"
+  libya.9.d: "In a monumental display of unity, thousands of Libyans march peacefully towards our capital, driven by an unwavering determination to make their voices heard. Their resolute purpose? To compel our nation to join Gaddafi's union. The sheer magnitude of their collective willpower leaves no doubt that their mission will succeed."
+  libya.9.f: "Revolution is when the masses make the revolution. A people’s revolution. But even when the revolution is made by others in the name of the masses, expressing what the masses want, it can be revolution. A people’s revolution because it has the support of the masses and interprets the will of the masses.\n― Muammar Gaddafi"
+  libya.9.a: "We have no choice"gaddafi_unification_current_target
```

- remove extra variable here? it was outside the double-quotes.
```suggestion
  libya.9.a: "We have no choice"
```